src/ui/js/Login.js

```diff
@@ -34,12 +34,7 @@ Login.getLoginHeader = function() {
       var welcomeText = 'Welcome to Button Men';
       if (Config.siteType == 'development') {
         $('#login_header').css('background-color', '#cccccc');
-        $('head').append(
-          $('<link>', {
-            'type': 'image/x-icon',
-            'rel': 'shortcut icon',
-            'href': '/dev_favicon.ico',
-          }));
+        $('#favicon').attr('href','/dev_favicon.ico');
```

Comment:
Nit: stylistically, i prefer horizontal whitespace between list items and string components in code, so `'href', '/dev_favicon.ico'` rather than `'href','/dev_favicon'`.

(A nit is something that i'm not going to block the pull over, but would like fixed if you have to make other changes and/or find it easy to do so.)